lark-stubs/lark.pyi

```diff
@@ -32,11 +32,25 @@ class LarkOptions:
     cache: Union[bool, str]
     g_regex_flags: int
     use_bytes: bool
+    import_paths: List[Union[str, Callable[[Union[None, str, PackageResource], str], Tuple[str, str]]]]
+    source_path: Optional[str]
+
+
+class PackageResource(object):
+    pkg_name: str
+    path: str
+    
+    def __init__(self, pkg_name: str, path: str):
+
+class FromPackageLoader:
+    def __init__(self, pkg_name: str, search_paths: Tuple[str, ...] = ...): ...
+    
+    def __call__(self, base_path: Union[None, str, PackageResource], grammar_path: str) -> Tuple[PackageResource, str]: ...
 
 
 class Lark:
-    source: str
-    grammar_source: str
+    source_path: str
+    source_code: str
```

Comment:
Sorry for not catching this earlier, but this should be `source_text`. Not everyone uses lark just to parse code.

Reply:
Well, it is the source code for the grammar. Maybe `source_grammar`?

Reply:
Hmm then my comment is irrelevant. But yes, maybe that's a better name anyway.